lynx/src/main/java/com/github/pedrovgs/lynx/LynxShakeDetector.java

```diff
@@ -56,17 +61,25 @@ public void init() {
   /**
    * Enables shake detector to open LynxActivity on shake.
    */
-  static void enable() {
+  public static void enable() {
     isEnabled = true;
   }
 
   /**
    * Disables shake detector to open LynxActivity on shake.
    */
-  static void disable() {
+  public static void disable() {
```

Comment:
These two public methods here are a smell telling us that this solution is not the best approach.